image.go

```diff
@@ -32,6 +32,8 @@ type Image interface {
 	Config(ctx context.Context) (ocispec.Descriptor, error)
 	// IsUnpacked returns whether or not an image is unpacked.
 	IsUnpacked(context.Context, string) (bool, error)
+	// ContentStore provides a content store which contains image blob data
+	ContentStore() content.Store
```

Comment:
This doesn't look right...

Where is this needed?

Reply:
See my previous 2 comments, `oci/spec_opts_unix.go` `WithImageconfig`

Reply:
Here: https://github.com/containerd/containerd/pull/1796/files#diff-f7b47b78a700db482d56ad8e9972e1e5R80

Reply:
Maybe this would make more sense with a better name? `ContentProvider()` ?

Reply:
This makes sense for convenience, but makes it a little hard to swap out for specialty content store. Perhaps we kick the can down the road on that one. May not seem obvious, but there are some feature and performance benefits to being able to swap this out.

I'm not against the change, however.

Reply:
Sounds good. Definitely a bit harder to swap it out, but also not impossible since it's just a 2 method interface to implement.